mailpoet/views/woo_system_info.html

```diff
@@ -0,0 +1,38 @@
+<table class="wc_status_table widefat" cellspacing="0">
+  <thead>
+  <tr>
+    <th colspan="3" data-export-label="MailPoet"><h2><%= __("MailPoet") %></h2></th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td data-export-label="Sending Method"><%= __("Sending Method:") %></td>
+    <td class="help">
+      <span class="woocommerce-help-tip" data-tip="<%= __("What method is used to sent out newsletters?") %>"></span>
+    </td>
+    <td><%= system_info.sending_method %></td>
+  </tr>
+  <tr>
+    <td data-export-label="Send all site's emails with"><%= __("Send all site's emails with:") %></td>
+    <td class="help">
+      <span class="woocommerce-help-tip" data-tip="<%= __("With which method are transactional emails sent?") %>"></span>
+    </td>
+    <td><%= system_info.transactional_emails %></td>
+  </tr>
+  <tr>
+    <td data-export-label="Task Scheduler method"><%= __("Task Scheduler method:") %></td>
+    <td class="help">
+      <span class="woocommerce-help-tip" data-tip="<%= __("What method controls the cron job?") %>"></span>
+    </td>
+    <td><%= system_info.task_scheduler_method %></td>
+  </tr>
+  <tr>
+    <td data-export-label="Cron ping URL"><%= __("Cron ping URL:") %></td>
+    <td class="help">
+      <span class="woocommerce-help-tip" data-tip="<%= __("The URL which needs to be pinged to get the cron started?") %>"></span>
```

Comment:
I would either remove the question mark on `The URL which needs to be pinged to get the cron started?` or turn the sentence into a question.

Reply:
3c901112c